src/main/Ui.java

```diff
@@ -0,0 +1,79 @@
+import java.util.ArrayList;
+
+public class Ui {
+    String message;
+    String divider;
+
+    public Ui() {
+        divider = "____________________________________________________\n";
+    }
+
+    public Ui(String message) {
+        this.message = message;
+        divider = "____________________________________________________\n";
+    }
```

Comment:
You can consider making the String divider into a constant, as it is being called in both constructors. As it is unchanged in each case, it might be a good idea to make it into a constant.